docs/using_zeronet/create_new_site.md

```diff
@@ -25,33 +26,33 @@ $ zeronet.py
 ...
 ```
 
-- This will create the initial files for your site inside ```data/13DNDkMUExRf9Xa9ogwPKqp7zyHFEqbhC2```.
+This will create the initial files for your site inside ```data/13DNDkMUExRf9Xa9ogwPKqp7zyHFEqbhC2```.
 
 > __Note:__
 > Windows users using the bundle version must browse into the ZeroBundle/ZeroNet folder and run `"../Python/python.exe" zeronet.py siteCreate`
 
 ### 2. Build/Modify site
 
 * Update the site files located in ```data/[your site address key]``` (eg: 13DNDkMUExRf9Xa9ogwPKqp7zyHFEqbhC2).
-* When your site is ready run:
+* When finished sign your site's content:
 
 ```bash
 $ zeronet.py siteSign 13DNDkMUExRf9Xa9ogwPKqp7zyHFEqbhC2
 - Signing site: 13DNDkMUExRf9Xa9ogwPKqp7zyHFEqbhC2...
 Private key (input hidden):
 ```
 
-* Enter the private key you got when you created the site. This will sign all files so peers can verify that the site owner is who made the changes.
+* Enter the private key you got when you created the site. This will sign all files so peers can verify that the site owner is the one who made changes.
```

Comment:
I think this would sound better if you changed "got" to "received" in the first sentence. I would also consider changing "site owner is the one who made the changes" to "site owner made the changes"